topics/single-cell/metadata.yaml

```diff
@@ -0,0 +1,49 @@
+---
+name: single-cell
+type: use
+title: Single Cell
+summary: Training material and practicals for all kinds of single cell analysis (particularly scRNA-seq!). When you generate your lovely gene lists for your cells, consider checking out our Transcriptomics tutorials for further network analysis!
+#docker_image: "quay.io/galaxy/transcriptomics-training"
+
+requirements:
+  -
+    type: "internal"
+    topic_name: introduction
+
+maintainers:
+  - bebatut
+  - mblue9
+  - heylf
+  - nomadscientist
```

Comment:
@nomadscientist should maintainers for single-cell topic just be you and @mtekman maybe?